connectors/src/connectors/webcrawler/temporal/activities.ts

```diff
@@ -372,14 +379,21 @@ export async function crawlWebsiteByConnectorId(connectorId: ModelId) {
   // checks for cancellation and throws if it's the case
   await Context.current().sleep(1);
 
-  if (pageCount > 0) {
-    await syncSucceeded(connector.id);
-  } else {
+  if (totalExtracted <= 0) {
+    await syncFailed(connector.id, "webcrawling_error_empty_content");
+  } else if (upsertedPageCount <= 0) {
+    // this means that every page was too big to be upserted
+    await syncFailed(connector.id, "webcrawling_error_content_too_large");
```

Comment:
same here, would go for a ratio; all pages too big are too restrictive
also woud use a var name `pagesTooBig` rather than `upsertedPages` to be clearer